portable/IAR/AtmelSAM7S64/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
+                                                unsigned int flag ) /* \arg  pio output write to be disabled */
+    {
         pPio->PIO_OWDR = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetOutputWriteStatus
-//* \brief Return PIO Output Write Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetOutputWriteStatus( // \return PIO Output Write Status
-        AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetOutputWriteStatus */
+/** \brief Return PIO Output Write Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetOutputWriteStatus( /* \return PIO Output Write Status */
+        AT91PS_PIO pPio )                                 /* \arg  pointer to a PIO controller */
+    {
         return pPio->PIO_OWSR;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_IsOutputWriteSet
-//* \brief Test if PIO OutputWrite is Set
-//*----------------------------------------------------------------------------
-__inline int AT91F_PIO_IsOutputWriteSet(
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  flag to be tested
-{
-        return (AT91F_PIO_GetOutputWriteStatus(pPio) & flag);
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetCfgPullup
-//* \brief Return PIO Configuration Pullup
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetCfgPullup( // \return PIO Configuration Pullup
-        AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_IsOutputWriteSet */
+/** \brief Test if PIO OutputWrite is Set */
+/**---------------------------------------------------------------------------- */
+    __inline int AT91F_PIO_IsOutputWriteSet( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                             unsigned int flag ) /* \arg  flag to be tested */
+    {
+        return( AT91F_PIO_GetOutputWriteStatus( pPio ) & flag );
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetCfgPullup */
+/** \brief Return PIO Configuration Pullup */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetCfgPullup( /* \return PIO Configuration Pullup */
+        AT91PS_PIO pPio )                         /* \arg  pointer to a PIO controller */
```

Comment:
Should this be one line?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)